### ISSUE TITLE
Wrong code snippet to copy dependency

### ISSUE THREAD

**Issue by sorinbrazdau** [OpenLiberty/docs#4517]
Document: Relational database connections with JDBC
http://openliberty.io/docs/21.0.0.6/relational-database-connections-JDBC.html

The code snippet used as an example to copy the dependency is wrong, gives an error: 

>  Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.3:dev (default-cli) on project: Unable to parse configuration of mojo io.openliberty.tools:liberty-maven-plugin:3.3:dev for parameter filter: Cannot find 'filter' in class io.openliberty.tools.maven.server.Dependency

```
<plugin>
<groupId>io.openliberty.tools</groupId>
<artifactId>liberty-maven-plugin</artifactId>
<version>3.3</version>
  <configuration>
    <copyDependencies>
      <dependency>
        <filter>com.ibm.db2:jcc:11.5.4.0</filter>
        <location>jdbc</location>
      </dependency>
    </copyDependencies>
  </configuration>
</plugin>
```

Instead it should be:

```
<plugin>
<groupId>io.openliberty.tools</groupId>
<artifactId>liberty-maven-plugin</artifactId>
<version>${liberty-maven-plugin.version}</version>
  <configuration>
    <copyDependencies>
      <dependencyGroup>
       	<location>jdbc</location>
     	  <dependency>
            <groupId>com.ibm.db2</groupId>
            <artifactId>jcc</artifactId>
            <version>11.5.4.0</version>
      	  </dependency>
      </dependencyGroup>
     </copyDependencies>
   </configuration>
</plugin>
```

**Comment by dmuelle** [comment]
Thanks for this feedback @sorinbrazdau. I've updated the draft with your suggestion (modified slightly per our style guidelines) and got @cherylking to verify the example. 

https://docs-draft-openlibertyio.mybluemix.net/docs/21.0.0.9/relational-database-connections-JDBC.html#_jdbc_driver_configuration_with_maven_or_gradle

update will publish to production with the 21.0.0.9 docs release

**Comment by mswatosh** [comment]
@dmuelle New article looks much better, there are a few things I noticed and some recommendations:

- I tried the new example with `<version>[3.3.4,)</version>` and it can't find the plugin, is that syntax valid inside the liberty maven plugin? Using `<version>3.3.4</version>` works fine. For reference I'm using Apache Maven 3.6.3. @cherylking maybe this is a question for you?

- The latest version of the jcc driver is 11.5.6.0, so lets use `<version>11.5.6.0</version>`
 
- For the line " In this example, the com.ibm.db2 dependency is copied to the jdbc directory " I think it would be better to say " In this example, the driver files from the `com.ibm.db2:jcc:11.5.6.0` dependency are copied to the jdbc directory " or just " In this example, the driver files are copied to the jdbc directory "

- Consider moving "However, the Open Liberty Gradle plug-in does not directly support ... " down to the Gradle section. When I first read through it I assumed we weren't providing a gradle example, because it continues into the Maven example

- Both of the times "Db2 JDBC driver" and "IBM Db2 JDBC driver" are mentioned I would just change to "driver".

One last thing, Liberty Maven Plugin vs Plug-in (and capitalized or not). I see each being used between different pieces of doc, maybe something that should be standardized?

**Comment by dmuelle** [comment]
Thanks @mswatosh - I've updated the doc per your recommendations:

https://docs-draft-openlibertyio.mybluemix.net/docs/21.0.0.9/relational-database-connections-JDBC.html#_jdbc_driver_configuration_with_maven_or_gradle

- [x] I tried the new example with <version>[3.3.4,)</version> and it can't find the plugin, is that syntax valid inside the liberty maven plugin? Using <version>3.3.4</version> works fine. **I introduced this error by copying the version syntax from the wrong part of the plug-in README. Updated as you suggested and verified with @cherylking**
- [x]   The latest version of the jcc driver is 11.5.6.0, so lets use <version>11.5.6.0</version>
- [x] For the line " In this example, the com.ibm.db2 dependency is copied to the jdbc directory " I think it would be better to say " In this example, the driver files from the com.ibm.db2:jcc:11.5.6.0 dependency are copied to the jdbc directory " or just " In this example, the driver files are copied to the jdbc directory "
- [x] Consider moving "However, the Open Liberty Gradle plug-in does not directly support ... " down to the Gradle section. When I first read through it I assumed we weren't providing a gradle example, because it continues into the Maven example
- [x] Both of the times "Db2 JDBC driver" and "IBM Db2 JDBC driver" are mentioned I would just change to "driver".
- [x] Liberty Maven Plugin vs Plug-in (and capitalized or not). I see each being used between different pieces of doc, maybe something that should be standardized? **Per style guidelines, it should consistently be "the Open Liberty Maven plug-in". The link text to the plug-in readme was slightly different to reflect the way it's written in the readme, which doesn't observe these guidelines. I changed it to be consistent with the rest of the doc here- thanks for the catch**
